serverless/functions/sync-webhook.protected.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+exports.handler = async function (context, event, callback) {
+  const common = require(Runtime.getAssets()['/common.js'].path);
+  const { response } = common(context, event, callback);
+  const client = context.getTwilioClient();
+
+  if (event.EventType === 'endpoint_disconnected') {
```

Comment:
Nice 👍🏼 

What other cases will this webhook cover?

* Removing a viewer for speakers and viewers

Any others?

Reply:
Yes removing viewers and speakers from sync. Probably cleaning up sync state is the main thing.

I think we probably want to keep using the room webhook for other clean up when the room is empty. I think the room has a 30 second connection timeout. Using sync for this would be more immediate but I think the room webhook is probably more reliable and simpler.

That might be it.